src/components/MultiHopTrade/components/MultiHopTradeConfirm/components/StepperStep.tsx

```diff
@@ -67,7 +67,11 @@ export const StepperStep = ({
   stepIndicatorVariant = 'default',
 }: StepperStepProps) => {
   const { indicator: indicatorStyles } = useStyleConfig('Stepper', {
-    variant: isError ? 'error' : stepIndicatorVariant,
+    variant: isError
+      ? stepIndicatorVariant === 'innerSteps'
```

Comment:
This is a little too specific for a generic component, but done in the name of backwards compatibility with the existing swapper. Will be cleaned up nicely once the new swapper is turned on.